### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by danielsarsi**
As GitHub Ribbon doesn't support high DPI, I changed to an alternative (pretty neat, btw).
http://tholman.com/github-corners/

### CODE REVIEW

**Comment by Gymnasiast** [comment]
... OK, first of all: sorry for letting this rot for almost five years.

We will try to review it soon-ish. Obviously, we need to look if everything is still applicable.

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta name="viewport" content="width=device-width, user-scalable=no">
   <title>OpenRCT2</title>
   <link rel="stylesheet" href="assets/css/index.css">
-  <script src="assets/js/index.js" type="text/javascript" async></script>
+  <script src="assets/js/index.js" type="text/javascript"></script>
```
Comment (Gymnasiast) [inline index.html:8]: I disagree with this. It makes the JS file a blocking resource. Please drop this commit.

File: index.html [hunk 2]
```diff
@@ -21,9 +21,9 @@ <h1>OpenRCT2</h1>
     <h2>Open source re-implementation of RollerCoaster Tycoon 2</h2>
     <ul id="buttons">
       <li>
-        <a class="link" href="https://github.com/OpenRCT2/OpenRCT2/releases/download/v0.0.3.1/OpenRCT2-0.0.3.1-windows.zip">Download Stable</a>
+        <a id="link" href="https://github.com/OpenRCT2/OpenRCT2/releases/download/v0.0.3.1/OpenRCT2-0.0.3.1-windows.zip">Download Stable</a>
         <div class="information">
-          <span class="version">0.0.3.1</span> — <a class="platform" href="#" title="Change platform…">Windows</a> — <span class="size">12.3 MB</span>
+          <span id="version">0.0.3.1</span> — <a id="platform" href="#" title="Change platform…">Windows</a> — <span id="size">12.3 MB</span>
```
Comment (Gymnasiast) [inline index.html:27]: I don't think there is really a good reason for doing this. Please also drop this commit.

**Comment by Gymnasiast** [comment]
I have rebased the branch, dropped any commits that were obsolete or not what we wanted and created a new PR: https://github.com/OpenRCT2/openrct2.github.io/pull/35

Sorry again.

**Comment by danielsarsi** [comment]
oh sorry didn't see it in time 😄

**Pull request closed without merging**